RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -2,33 +2,42 @@
 @using System.Globalization
 @inject IBillRepository billRepository
 
-@section Styles {
-<link rel="stylesheet" href="/css/BillDetails.css" />
-}
 @{
     var billId = Guid.Parse(RouteData.Values["guid"].ToString());
```

Comment:
Since we already gracefully handle missing bills, we should also use TryParse instead of Parse on the Guid.